python/astro_metadata_translator/headers.py

```diff
@@ -283,28 +346,10 @@ class or else support automatic translation class determination.
     paths.extend(translator.search_paths())
 
     # Prioritize file system overrides
-    for p in paths:
-        correction_file = os.path.join(p, target_file)
-        if os.path.exists(correction_file):
-            with open(correction_file) as fh:
-                log.debug("Applying header corrections from file %s", correction_file)
-                corrections = yaml.safe_load(fh)
+    modified = _find_from_file(header, paths, target_file)
 
-                # Apply corrections
-                header.update(corrections)
-
-                return True
+    # Apply updates from resources only if none found in files
+    if not modified:
+        modified = _find_from_resource(header, *translator.resource_root(), target_file)
```

Comment:
Would be clearer if you did

    package, resource_root = translator.resource_root()
    modified = _find_from_resource(header, package, resource_root, target_file)

And I think it would make any errors from potential future API changes easier to understand.